src/app/connections/list/list.component.html

```diff
@@ -5,9 +5,9 @@
         <div class="card-pf card-pf-view card-pf-view-xs">
           <div class="card-pf-body">
 
-            <!-- Card Dropdown -->
-
+            <!-- Card Heading -->
             <div class="card-pf-heading-kebab">
+              <!-- Card Actions --
```

Comment:
Ah that was to comment out the section. Would you like for me to change it to have a separate commented out block from the title?